### PR TITLE
tests for update and create

### PR DESCRIPTION
**Pull request by puneeth2001**

### CODE REVIEW

File: vulnerabilities/data_dump.py [hunk 1]
```diff
@@ -34,12 +34,12 @@ def debian_dump(extract_data):
     Save data scraped from Debian' security tracker.
     """
     for data in extract_data:
-        vulnerability = Vulnerability.objects.create(
+        vulnerability = Vulnerability.objects.update_or_create(
```
Comment (singh1114) [inline vulnerabilities/data_dump.py]: I think `update_or_create` returns more than one thing.

Comment (singh1114) [reply vulnerabilities/data_dump.py]: That is why this test is failing.

File: vulnerabilities/data_dump.py [hunk 2]
```diff
@@ -34,12 +34,12 @@ def debian_dump(extract_data):
...
             summary=data.get('description', ''),
         )
         VulnerabilityReference.objects.create(
             vulnerability=vulnerability,
-            reference_id=data.get('vulnerability_id', ''),
+            reference_id=data.get('vulnerability_id', '')
```
Comment (singh1114) [inline vulnerabilities/data_dump.py]: This should check if the object was created at the last stage or not.

File: vulnerabilities/tests/test_data_dump.py [hunk 3]
```diff
@@ -63,8 +63,10 @@ def test_debian_data_dump(self):
         self.assertTrue(Vulnerability.objects.get(
                         summary='Multiple stack-based buffer overflows in mimetex.cgi in mimeTeX'))
 
-        self.assertTrue(Vulnerability.objects.get(
+        self.assertFalse(Vulnerability.objects.get(
```
Comment (singh1114) [inline vulnerabilities/tests/test_data_dump.py]: You should not be changing the full meaning of the test.

**Comment by pombredanne** [comment]
@singh1114 @puneeth2001 is this PR still alive?

**Comment by pombredanne** [comment]
@puneeth2001 Thank you for your efforts... since you did not reply, I am closing this for now. Please reopen as needed if you want to complete this.

**Pull request closed without merging**